src/table/header-cell/index.tsx

```diff
@@ -49,6 +50,7 @@ export function TableHeaderCell({
   updateColumn,
   resizableColumns,
   onResizeFinish,
+  __isEditable,
```

Comment:
I thought it was a good idea to make it internal to discourage users from directly using the component. But, since this prop only renders an icon, I've made it public now